### ISSUE TITLE
display of onlyInteger message requires numericality message be defined

### ISSUE THREAD

**Issue by drewnichols** [DockYard/ember-validations#229]
When using a validation defined as:
```
    someCoolProperty: {
      numericality: {
        greaterThanOrEqualTo: 1,
        lessThanOrEqualTo: 12,
        onlyInteger: true,
        messages: {
          greaterThanOrEqualTo: "cool property must be larger or equal to 1",
          lessThanOrEqualTo: "cool property must be smaller or equal to 12",
          onlyInteger: "no fractions dude"
        }
      }
```

none of the messages defined in the messages hash display. Only the default messages display.

When defined as:
```
    someCoolProperty: {
      numericality: {
        greaterThanOrEqualTo: 1,
        lessThanOrEqualTo: 12,
        onlyInteger: true,
        messages: {
          greaterThanOrEqualTo: "cool property must be larger or equal to 1",
          lessThanOrEqualTo: "cool property must be smaller or equal to 12",
          onlyInteger: "no fractions dude",
          numericality: "loan duration must be between 1 and 12 months"
        }
      }
```

the custom messages are displayed

**Comment by tgreiser** [comment]
FYI, this is fixed in 2.0 branch.

https://github.com/dockyard/ember-validations/commit/38eb27248d6c437749cf05a9c24114efdf9d6f5c

**Comment by martndemus** [comment]
I am going to close this issue for now, feel free to reopen this issue if you feel like it's still relevant and we should take a look at it.
